apps/website/app/page.tsx

```diff
@@ -168,6 +170,8 @@ const Home = () => {
           </div>
         </div>
       </div>
+      <MobileStickyFooter />
+      <DesktopFooter />
```

Comment:
Müssen die nicht in die layout.tsx? Die sollen doch auf jeder Seite dargestellt werden?